config/thermofloor/heatit20a.xml

```diff
@@ -0,0 +1,225 @@
+<!--
+Heatit Z-Water
+https://products.z-wavealliance.org/products/3102
+--><Product Revision="3" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="ProductPic">images/thermofloor/heatit20a.png</MetaDataItem>
+    <MetaDataItem id="020a" name="ZWProductPage" type="0003">https://products.z-wavealliance.org/products/3102/</MetaDataItem>
+    <MetaDataItem name="InclusionDescription">Adding the device:
+When the primary controller is set to add/remove modus press the ”INCLUSION” button once on the device
+    <MetaDataItem id="020a" name="Identifier" type="0003">Z-Water</MetaDataItem>
+    <MetaDataItem name="Description">Heatit Z-Water is equipped with 10 relay outputs and 4 analog inputs, and a Z-Wave radio for interfacing to the wireless
+Z-Wave network.
+The regulator can be power supplied from a 230V AC mains connection, and is able to deliver an output supply of 24V DC.
+Heatit Z-Water relay outputs are able to be freely controlled from the Z-Wave network, and can be used for several
+purposes, e.g. on/off control of light, control of valve actuators for an underfloor heating system, or control of other home
+automation systems.
+Heatit Z-Water inputs are analog inputs for interfacing simple temperature sensors; NTC, PT1000, etc.
+It is possible to configure the level and the indication of the status indicator LED in the front of the Heatit Z-Water
+regulator./MetaDataItem>
+    <MetaDataItem name="ProductManual">https://Products.Z-WaveAlliance.org/ProductManual/File?folder=&filename=Manuals/3102/Manual_Heatit_Z-Water_analog%20inputs_FW%202.1_Ver2019-A_ENG.pdf</MetaDataItem>
+    <MetaDataItem name="ExclusionDescription">Removing the device:
+When the primary controller is set to add/remove modus press the ”INCLUSION” button once on the device
+    <MetaDataItem id="020a" name="FrequencyName" type="0003">CEPT (Europe)</MetaDataItem>
+    <MetaDataItem name="ResetDescription">By pressing buttons Right and Center (down and confirm) for 20 seconds, the thermostat will perform a complete factory reset.
+NB! Please use this procedure only when the primary controller/ gateway is missing or otherwise inoperable.</MetaDataItem>
+    <MetaDataItem name="Name">Heatit Z-Water</MetaDataItem>
+    <ChangeLog>
+      <Entry author="Espen Throndsen - espenthrondsen@gmail.com" date="21 Oct 2019" revision="1">Initial Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/3102/xml</Entry>
+      <Entry author="Lars Tobias Skjong-Børsting - larstobi@relatime.no" date="28 Jun 2020" revision="1">Added ProductPic</Entry>
+    </ChangeLog>
+  </MetaData>
+	<CommandClass id="32" name="COMMAND_CLASS_BASIC" version="1" request_flags="5" mapping="37">
```

Comment:
This should not be required. It should be auto discovered.

Reply:
Removed